operator-framework-core/src/main/java/io/javaoperatorsdk/operator/processing/event/rate/PeriodRateLimiter.java

```diff
@@ -58,4 +57,23 @@ public Optional<Duration> acquirePermission(ResourceID resourceID) {
   public void clear(ResourceID resourceID) {
     limitData.remove(resourceID);
   }
+
+  @Override
+  public void initFrom(LimitingRateOverPeriod configuration) {
+    this.refreshPeriod = Duration.of(configuration.within(),
+        configuration.unit().toChronoUnit());
+    this.limitForPeriod = configuration.maxReconciliations();
+  }
+
+  public boolean isActivated() {
```

Comment:
Don;t think this needs to be exposed, thus should not be public.

Reply:
It's used in a test in a different package so no choice… :(